### PR TITLE
this commit fixes #6

### LINKED ISSUE #6: User Authentication
## Endpoints
Endpoints are provided by [bigfastapi](https://bigfa.st/docs/socialauth)

## Example
```python
from bigfastapi.auth import app as auth
 
app.include_router(auth, tags=[Auth])
```

### PR DESCRIPTION
**Pull request by blacdev**
It also covers issues with modules not included in the Bigfast build
closes #6

### CODE REVIEW

**Comment by blacdev** [comment]
@phemmylintry

**Comment by blacdev** [comment]
@nseetim @odizee can i get a reason why this has not been merged yet?

**Comment by samsoft00** [comment]
Hey @blacdev , your MR look strange, why do you update requirement file? Do you read readme file on how to set this up?

**Comment by blacdev** [comment]
The requirements file covers all the modules/library needed in bigfast. This is because i am currently one of the contributors and i know and have experienced issues with not having all the modules installed. That is why i added it @samsoft00

**Comment by samsoft00** [comment]
I understand you @blacdev, but I think we can fix any issue right from bigfat. 

Also, things seems to be working perfectly from my side, I didn't install any other package aside the basic requirements.

@phemmylintry can you help us look into this?

**Comment by blacdev** [comment]
Okay then. I'll take them out now. @samsoft00

**Comment by phemmylintry** [comment]
@samsoft00 @blacdev 
I noticed some packages are still missing even when I installed bigfast directly, I don't know why that is but it doesn't hurt to have all the requirements there just to be double sure. bigfast itself is currently ongoing changes every time. So we don't break production. Thanks

**Comment by blacdev** [comment]
@phemmylintry i'll add it back. Do i also add a better env.sample file that covers all env variables?

**Comment by phemmylintry** [comment]
@blacdev Error from google when testing with keys

image

File: .env.sample [hunk 1]
```diff
@@ -20,4 +21,8 @@ FILES_BASE_FOLDER="filestorage"
 TEMPLATE_FOLDER="templates"
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
-REDIRECT_URL=
\ No newline at end of file
+GOOGLE_SECRET=
+API_REDIRECT_URL=
+REDIRECT_URL=
+BASE_URL=
+LANDING_PAGE_FORM_PATH=
```
Comment (phemmylintry) [inline .env.sample:28]: EOL

**Pull request closed without merging**